src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.TargetingPackResolution.targets

```diff
@@ -55,17 +55,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       -->
       
       
-      <!-- KnownAppHost items will look something like this:
-      
```

Comment:
This is not meant to be uncommented.

Reply:
It comes from the .NET Core SDK bundled versions props file.

Reply:
Yes, just noticed it while looking at the list of changes. I just submitted a new commit to revert this